src/components/cards/DashBaordUpcommingEventCard.tsx

```diff
@@ -0,0 +1,114 @@
+import { View, StyleSheet } from "react-native";
+import { Card, Text } from "react-native-paper";
+import React from "react";
+import {
+  responsiveHeight,
+  responsiveWidth,
+  responsiveFontSize,
+} from "react-native-responsive-dimensions";
+import { BlurView } from "expo-blur";
+import { DashBaordUpcommingEventCardInterface } from "../../interface/cards/DashBaordUpcommingEventCardInterface";
+import { Colors} from "../../components/theme/theme"
+
+export const DashBaordUpcommingEventCard: React.FC<
+DashBaordUpcommingEventCardInterface
+> = ({
+  width = 100,
+  height = 70,
+  eventName = '',
+  textColor = Colors.textColor,
+  eventDate = '',
+  companyName = '',
+  borderRadius = 8,
+  imageSource = { uri: 'https://picsum.photos/700' },
```

Comment:
Create assets folder and store the reuired image. Then pass image path here. Dont not pass internet url. @rohan-arya-w  Please confirm